### PR TITLE
Adding Flow Bin

### PR DESCRIPTION
**Pull request by greensirius**
I added new package for type checking using flow version ^.0.78.0
- I ve got a lot of issue when I setup this package, like

### CODE REVIEW

**Comment by nelvson** [comment]
LGTM!

**Comment by parelhutahaean** [comment]
This is a comment.

File: junk.js [hunk 1]
```diff
@@ -0,0 +1 @@
+this is just a text
```
Comment (nelvson) [inline junk.js]: Should be in a comment, e.g.
```jsx
//this is just a text
```

Comment (andreirawan97) [inline junk.js]: What

Comment (RaymondAnggara) [reply junk.js]: delete this

Comment (nelvson) [reply junk.js]: Should be in a comment, e.g.
```jsx
//this is just a text
```

Comment (RuneXa) [inline junk.js]: _|___|_  ╰(º o º╰)

Comment (cindmichelle) [reply junk.js]: ewww

Comment (andreirawan97) [reply junk.js]: //What

**Pull request merged**